test/AutoSubscribe.spec.tsx

```diff
@@ -566,3 +567,24 @@ describe('AutoSubscribe', function() {
 
     runTests(makeComponent);
 });
+
+it('Test hook system', function() {
+    SimpleStoreInstance = new SimpleStore();
+
+    function FuncComp(): JSX.Element {
+        const val = SimpleStoreInstance.getDataSingleKeyed();
+        return <>{ val.toString() }</>;
```

Comment:
I hate that TS makes you do this. Function components should be allowed to return strings directly. :(